tests/memealyzer.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect, WebSocket } from "@playwright/test";
+
+var uri: string = "http://localhost:1080/";
+
+interface Image {
+  id: string;
+  Id: string;
+}
+
+interface ImageEvent {
+  arguments: Image[];
+}
+
+test("Add Meme Test", async ({ page }) => {
```

Comment:
What are the expectations for this test? what are we testing? It can be in a comment